src/tests/ftest/nvme/nvme_pool_capacity.yaml

```diff
@@ -46,4 +44,4 @@ ior:
     #    - [4000000000, 18000000000, 1000000, 3000000000, FAIL]         #[4G, 18G, 1M, 3G, FAIL]
         - [300000000000, 550000000000, 1000000000, 8000000000, PASS]          #[300G, 550G, 1G, 8G, PASS]
     #  Blocked by DAOS-4803
-    #    - [300000000000, 550000000000, 1000000000, 800000000000, FAIL]         #[300G, 550G, 1G, 80G, FAIL]
```

Comment:
Is the removal of the newline at the end of the file intentional?

Reply:
I did not removed it intentionally

Reply:
I am not sure why it got removed but don't think it will matter for execution